stdlib/builtin/rb_config.rbs

```diff
@@ -12,7 +12,7 @@ module RbConfig
   #
   def self.expand: (String val, ?Hash[String, String] config) -> String
 
-  def self.fire_update!: (String key, String val, ?Hash[String, String] mkconf, ?Hash[String, String] conf) -> Array[String]
+  def self.fire_update!: (String key, String val, ?Hash[String, String] mkconf, ?Hash[String, String] conf) -> (Array[String] | nil)
```

Comment:
You can write `Array[String]?` instead of `| nil`. (I usually write `T?` instead of `T | nil`.)

Reply:
I fix by https://github.com/ruby/ruby-signature/pull/233/commits/eae3284bf88944096f1fa33b1283cd3de6e708d0.